Makefile

```diff
@@ -91,9 +91,9 @@ $(HEADS_MAKE): $(build)/$(make_dir)/.configured
 
 # Once we have a proper Make, we can just pass arguments into it
 all linux cpio run: $(HEADS_MAKE)
-        LANG=C MAKE=$(HEADS_MAKE) $(HEADS_MAKE) $(MAKE_JOBS) $@
+        LANG=C MAKE=$(HEADS_MAKE) $(MAKE_JOBS) $@
```

Comment:
Does this work? I think you might have removed wrong part of the string (`$(HEADS_MAKE)` instead of `MAKE=$(HEADS_MAKE)`).

Reply:
@SergiiDmytruk the line 94 shows `LANG=C MAKE=$(HEADS_MAKE) $(MAKE_JOBS) $@`
I basically changed:         
`LANG=C MAKE=$(HEADS_MAKE) $(HEADS_MAKE) $(MAKE_JOBS) $@` 
for 
`LANG=C MAKE=$(HEADS_MAKE) $(MAKE_JOBS) $@`

Since originally there was two `$(HEADS_MAKE) $(HEADS_MAKE)` passed. No harm, but make logs under build/logs/* were showing that double `/build/make-4.2.1/make` everywhere.

Reply:
@tlaurion I probably miss something. It's just that `LANG/MAKE=...` sets environment variables and `$(MAKE_JOBS)` expands to `-j...` which is what should be interpreted as a command in this case, I just don't see how it can execute `make` at the end.

Reply:
Now I see your point. Actually, I do not even understand why it was put there. Was removed under https://github.com/osresearch/heads/pull/1128/commits/790671f1259e288f771233105c973b70c090c3ac and commented at https://github.com/osresearch/heads/pull/1128#issuecomment-1059491383